src/styles/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: calendarWidth($cell-day-size, $num-of-column, 0);
+
+  &__day {
+    @include square($cell-day-size);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    background-color: $bg-day-color;
+    border: 1px solid #000;
+
+    transition: background-color 0.5s, transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
This can be written under general selector (not `:hover`)

Reply:
Thx for review ))